### ISSUE TITLE
Consider Updates to Attestor Type Documentation

### ISSUE THREAD

**Issue by Strakeln** [in-toto/witness#372]
There doesn't seem to be a place where the attestor type names are specified (for input with `witness run -a`). The attestor types have documentation pages found [here](https://github.com/in-toto/witness/tree/main/docs/attestors), but the actual argument provided to the `-a` option is not easily discovered.

For example, the CommandRun attestor documentation can be found [here](https://github.com/in-toto/witness/blob/main/docs/attestors/commandrun.md), but the actual argument provided to `-a`, `command-run`, is not easily found. This may be a poor example as there appear to be separate issues with specifying `-a command-run`.

**Comment by tannerjones4075** [comment]
Hello @Strakeln thanks for brining this to our attention. We have a [doc](https://github.com/in-toto/witness/blob/main/docs/commands.md) file that outlines the witness arguments. Is this what you are referencing?

**Comment by Strakeln** [comment]
Hey @tannerjones4075 - that is an example of what I'm referring to. Specifically looking at the following option:

` -a, --attestations strings                          Attestations to record ('product' and 'material' are always recorded) (default [environment,git])`

The valid `strings` arguments to provide to this option are not easily found. The help text mentions `product`, `material`, `environment`, and `git`. I believe that [this page](https://github.com/in-toto/witness/tree/main/docs/attestors) lists the available attestors, but not the specific string we should provide to the `-a` option.

I realize that some of the specific strings are the same as the document name in the page I linked (ex: product, material, jwt, oci, environment, git, maven), but I believe there is at least one example where that is not the case (commandrun, for which I believe the string is actually `command-run`. 

Basically, I'm just looking for a list of valid values to provide to the `-a` option.

**Comment by matglas** [comment]
Hi @Strakeln would you be able to provide your feedback on the new `witness attestors list` command.

**Comment by Strakeln** [comment]
Hey @matglas - Output of the command looks great, exactly what I was looking for, thanks!

**Comment by matglas** [comment]
Great! Thank you for the response. Could you close the ticket as Completed?
